src/lib.rs

```diff
@@ -79,14 +98,43 @@ pub fn run(args: Args) {
                 args.filter_arg.clone(),
             );
 
-            SearcherBuilder::new()
-                .multi_line(true)
-                .build()
+            get_searcher(output_mode)
                 .search_path(&matcher, path, printer.sink_with_path(&matcher, path))
                 .unwrap();
         });
 }
 
+fn format_relative_path(path: &Path, output_mode: OutputMode) -> &Path {
+    if output_mode == OutputMode::Vimgrep && path.starts_with("./") {
+        path.strip_prefix("./").unwrap()
+    } else {
+        path
+    }
+}
+
+fn get_printer(output_mode: OutputMode) -> grep::printer::Standard<termcolor::NoColor<io::Stdout>> {
+    match output_mode {
+        OutputMode::Normal => grep::printer::Standard::new_no_color(io::stdout()),
+        OutputMode::Vimgrep => grep::printer::StandardBuilder::new()
+            .per_match(true)
```

Comment:
These options (and the additional option to the `SearcherBuilder` below) were from looking for references to the `vimgrep` option in the `ripgrep` source code, some of them may already be the default value